core/src/main/kotlin/com/alecstrong/sql/psi/core/postgresql/PostgreSql.bnf

```diff
@@ -101,6 +105,18 @@ boolean_data_type ::= 'BOOLEAN' | 'BOOL'
 
 json_data_type ::= 'JSON'
 
+with_clause ::= WITH [ RECURSIVE ] {cte_table_name} AS '(' {with_clause_auxiliary_stmt} ')' ( ',' {cte_table_name} AS '(' {with_clause_auxiliary_stmt} ')' ) * {
```

Comment:
i might be missing it but it looks like this is the same rule definition as sql.bnf#with_clause so maybe only the auxiliary is needed?

Reply:
Yep you're right! I was doing some weird stuff before I introduced `with_clause_auxiliary_stmt` so this was just a remnant of that.

Fixed in https://github.com/AlecStrong/sql-psi/pull/170/commits/21ff9d6d4d472234d003054942653ce998137860.